<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDelayer_Filebox</class>
 <widget class="QWidget" name="PasswordDelayer_Filebox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="file_groupBox">
     <property name="title">
      <string>Fichier</string>
     </property>
     <layout class="QVBoxLayout" name="file_layout">
      <item>
       <layout class="QHBoxLayout" name="path_layout">
        <item>
         <widget class="QLabel" name="path_label">
          <property name="text">
           <string>Adresse :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="path_lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="path_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="openFileMode_layout">
        <item>
         <widget class="QPushButton" name="openFile_button">
          <property name="text">
           <string>Ouvrir fichier mot de passe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createFile_button">
          <property name="text">
           <string>Cr√©er fichier mot de passe</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="passwordMode_layout">
        <item>
         <widget class="QPushButton" name="getPassword_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Donner mot de passe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetPassword_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Changer mot de passe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeFile_button">
          <property name="text">
           <string>Fermer fichier</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
